File: Leetcode/Easy Problems/1672. Richest Customer Wealth/dec-02-2023.js
/*
  December 02, 2023
  - Runtime Achieved: 54 ms 
  - Memory Achieved: 41.8 MB
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let m = accounts;
    let total = 0;
    for(let i = 0;i<m.length;i++)
    {
        let n = accounts[i];
        temp = 0;
        for(let j = 0;j<n.length;j++) 
        {
            temp += n[j];
            if(temp > total) {total = temp;}
        }
    }
    return total;
};

File: Leetcode/Easy Problems/1480. Running Sum of 1d Array/dec-02-2023.py
# December 02, 2023
# - Runtime Achieved: 44 ms
# - Memory Achieved: 16.5 MB
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        #arr = [1] * len(nums)
        for x in range(1,len(nums)): nums[x]+=nums[x-1]
        return nums
